package.json

```diff
@@ -24,6 +24,10 @@
             "url": "http://davidberner.co.uk/"
         }
     ],
+    "scripts": {
+        "compile": "node-sass src/shell.scss",
+        "lint": "stylelint 'src/**.scss'"
```

Comment:
Do we wanna run this over `/test` and `/doc` to to save having to correct linting issues in PR's? Mainly `/test` seeing as that is part of `CONTRIBUTING.md`.